README.md

```diff
@@ -147,18 +153,18 @@ fpath = (my/completions $fpath)
 
 #### Installation Issues
 
-- If you are trying to install as non-root user (`npm install -g tldr`) and get something like - 
- 
+- If you are trying to install as non-root user (`npm install -g tldr`) and get something like -
```

Comment:
I know it's not related to the changes in this PR, but the appropriate punctuation here would be `:`, not `-`. Since you're already editing these lines, maybe fix that too.